tank_battalion/enemy.pde

```diff
@@ -19,7 +19,7 @@ class Enemy {
   void update(int[][] grid_nodes) {
     int target_x = x;
     int target_y = y;
-    int direction = int(random(0, 4));
+    int direction = -1; //int(random(0, 4));
```

Comment:
I'm wondering if this is intentional?

Reply:
Yes, Rijk mentioned in the issue "Note that the enemies shouldn't move yet. We will implement that later.". This was an easy way to disabling movement without removing code which we could reuse.

Reply:
Excellent! I should have better read the issue.